Mesh_3/include/CGAL/Mesh_3/polylines_to_protect.h

```diff
@@ -25,10 +25,37 @@
 #include <map>
 #include <CGAL/tuple.h>
 #include <CGAL/Image_3.h>
-#include <CGAL/internal/Mesh_3/split_in_polylines.h>
+#include <CGAL/boost/graph/split_graph_into_polylines.h>
 #include <CGAL/internal/Mesh_3/Graph_manipulations.h>
 #include <boost/graph/adjacency_list.hpp>
 
```

Comment:
@sloriot Please write comments *below* the code and not *above* it. The mails that are send by Github are strange if you write comments above the code.